src/encoded/static/components/search.js

```diff
@@ -345,6 +345,20 @@ const ExperimentComponent = (props, reactContext) => {
         );
     }
 
+    // Get SCREEN link and FactorBook link if they exist
+    const screenLink = result.dbxrefs.filter(dbxref => (dbxref.indexOf('SCREEN') > -1))[0];
```

Comment:
can dbxrefs ever be falsey? If yes, this will throw an exception.

Reply:
I don't think so, but I'll account for that just in case.